src/test/java/hudson/plugins/git/SCMTriggerTest.java

```diff
@@ -277,12 +277,9 @@ private String prepareRepo(ZipFile repoZip) throws IOException {
     private Future<Void> triggerSCMTrigger(final SCMTrigger trigger)
     {
         if(trigger == null) return null;
-        Callable<Void> callable = new Callable<Void>() {
-            public Void call() throws Exception
-            {
-                trigger.run();
-                return null;
-            }
+        Callable<Void> callable = () -> {
+            trigger.run();
+            return null;
         };
         return singleThreadExecutor.submit(callable);
```

Comment:
or

```java
return singleThreadExecutor.submit(() -> trigger.run());
```

Reply:
I'm not yet skilled enough with the Java 8 changes to adapt to this message:

> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.1:testCompile (default-testCompile) on project git: Compilation failure
> [ERROR] /home/mwaite/git/jenkins/git-plugin/src/test/java/hudson/plugins/git/SCMTriggerTest.java:[280,43] incompatible types: java.util.concurrent.Future<capture#1 of ?> cannot be converted to java.util.concurrent.Future<java.lang.Void>

Any hints?